### PR TITLE
Add Director module support, ssh_ressource support, charset db support

### PR DESCRIPTION
**Pull request by plarivee**
This adds
- ssh remote command for monitoring module
- ssh_resource addition
- charset support for database resource
- Director module support
- Adds the mysql-client requirement for Director module schema

### CODE REVIEW

**Comment by plarivee** [comment]
ok, I'll fix the code for the Lint check.

File: manifests/params.pp [hunk 1]
```diff
@@ -106,6 +106,7 @@
         'php5-ldap',
         'php5-mysql',
         'php5-pgsql',
+        'mysql-client-core-5.5',
```
Comment (arioch) [inline manifests/params.pp]: This is something that conflicts with a number of mysql modules.

**Comment by arioch** [comment]
First of all thanks for your contribution.

Could you split the features you add to separate branches and pull requests?
One for ssh_resource, one for director, ...

If you need any advice we're here to help you getting your PR merged.

File: manifests/params.pp (hunk 1 above)
Comment (plarivee) [inline manifests/params.pp]: I'll make it a parameter to install it or not.

**Comment by plarivee** [comment]
Yes I can split. Will redo.

File: manifests/params.pp (hunk 1 above)
Comment (arioch) [inline manifests/params.pp]: Better yet use a module that provides this package.
I can recommend this one: [puppetlabs/mysql](https://forge.puppet.com/puppetlabs/mysql).

Comment (plarivee) [inline manifests/params.pp]: yeah, not fond using a module to provide only a client. I use the mysql module to manage my db servers.

Comment (arioch) [inline manifests/params.pp]: Nobody forces you to use the server part of the module.
I can't speak for every module but at least with aforementioned one you can ```include mysql::client``` to have only the client binaries installed.

Comment (plarivee) [inline manifests/params.pp]: yes I see the advantage of your point for cross platform.  From my side, I don't like / see the advantage of using a  module to install a simple client that is available already in a package. Then it's one less module to care about. 
That is why I mentioned the param way of installing or not a package. This  will make things simple I think.

**Pull request closed without merging**